### PR TITLE
Add 1.8.0 release blogpost

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Relates to https://github.com/kubewarden/helm-charts/issues/310

### CODE REVIEW

**Comment by viccuad** [comment]
This shouldn't be merged before releasing 1.8.0

File: content/blog/2023/10/kubewarden-1.8-release.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+For Kubewarden, this means we should expect more users to configure and fine
+tune the deployment to fit their stacks.
+
+## Changes
+
+With 1.8.0, the Kubewarden Controller now separately reconciles the configuration of both
+tracing and metrics for PolicyServers. In the past, enabling metrics in
+Kubewarden meant enabling both, and now users can decide to enable each separately, as we
+have realized from the requests raised in GitHub.
```
Comment (fabriziosestito) [inline content/blog/2023/10/kubewarden-1.8-release.md:32]: maybe `in GitHub` can be rephrased with `from the community`

File: content/blog/2023/10/kubewarden-1.8-release.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+We have made the concious decision to make this a breaking change in the helm
+chart `values.yaml`, which we never do.
```
Comment (fabriziosestito) [inline content/blog/2023/10/kubewarden-1.8-release.md:35]: maybe we could replace `we never do` with something like `it is uncommon`

File: content/blog/2023/10/kubewarden-1.8-release.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    jaeger:
+      endpoint: "my-open-telemetry-collector.jaeger.svc.cluster.local:4317"
+      tls:
+        insecure: true
+```
+
+## Configuring OTLP manually
+
+As always, users can decide instead to configure metrics and/or tracing support on
+their own. For example in case that they want only some of the Kubewarden
```
Comment (fabriziosestito) [inline content/blog/2023/10/kubewarden-1.8-release.md:95]: Users may still configure metrics and/or tracing support themselves, as desired. 
or
Users have the option to manually configure metrics and/or tracing support.

**Review by fabriziosestito — APPROVED** [review]
overall LGTM, left some suggestions here and there

File: content/blog/2023/09/audit-scanner-feature.md
Comment (flavio) [inline content/blog/2023/09/audit-scanner-feature.md:1]: This should not be part of this PR

File: content/blog/2023/10/kubewarden-1.8-release.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+date: 2023-10-11
+---
+
+Today we are happy to announce the release of Kubewarden 1.8.0! 🎉🥳
+
+This is a small release, focused on [OpenTelemetry](https://opentelemetry.io/).
+
+The OpenTelemetry Protocol (OTLP) got its [first 1.0.0
+version](https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v1.0.0)
+in July 2023, and the OpenTelemetry stack is becoming more stable; several
```
Comment (flavio) [inline content/blog/2023/10/kubewarden-1.8-release.md:14]:
```suggestion
in July 2023; several
```

It seems a bit contradictory/confusing to read on one line that otel is becoming more stable, but then on the next paragraph to read the entire opposite. So either we use a synonym, or we just drop that like in the suggestion from above.

File: content/blog/2023/10/kubewarden-1.8-release.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+libraries got their first 1.0.0 release, such as the [Go metric
+SDK](https://opentelemetry.io/blog/2023/otel-go-metrics-sdk-stable/) or the
+[.NET Automatic
+Instrumentation](https://opentelemetry.io/blog/2023/otel-dotnet-auto-instrumentation/).
+
+Still, the OTLP stack is not yet stable, and unannounced backwards-incompatible
```
Comment (flavio) [inline content/blog/2023/10/kubewarden-1.8-release.md:20]:
```suggestion
Still, the OpenTelemetry stack is not yet stable, and unannounced backwards-incompatible
```

I prefer to avoid acronyms that are probably not known by the average reader

File: content/blog/2023/10/kubewarden-1.8-release.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+changes still happen. You can have a look at the status of each of their
+libraries and protocols [here](https://opentelemetry.io/status).
+
+For Kubewarden, this means we should expect more users to configure and fine
+tune the deployment to fit their stacks.
+
+## Changes
+
+With 1.8.0, the Kubewarden Controller now separately reconciles the configuration of both
+tracing and metrics for PolicyServers. In the past, enabling metrics in
```
Comment (flavio) [inline content/blog/2023/10/kubewarden-1.8-release.md:30]:
```suggestion
tracing and metrics for Policy Server instances. In the past, enabling metrics in
```

File: content/blog/2023/10/kubewarden-1.8-release.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+    port: 8080
+  tracing:
+    enabled: True
+    jaeger:
+      endpoint: "my-open-telemetry-collector.jaeger.svc.cluster.local:4317"
+      tls:
+        insecure: true
+```
+
+## Configuring OTLP manually
```
Comment (flavio) [inline content/blog/2023/10/kubewarden-1.8-release.md:92]:
```suggestion
## Configuring OpenTelemetry manually
```

**Review by flavio — CHANGES_REQUESTED** [review]

**Review by fabriziosestito — APPROVED** [review]

File: content/blog/2023/10/kubewarden-1.8-release.md (hunk 2 above)
Comment (jhkrug) [reply content/blog/2023/10/kubewarden-1.8-release.md:35]: Yeah, 'we never do' doesn't work as are doing it. How about 'which we rarely do'?

File: content/blog/2023/09/audit-scanner-feature.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+  - Víctor Cuadrado
+date: 2023-10-05
+---
+
+New with the already released Kubewarden `v1.7.0` stack, we welcome a new module:
+the Audit Scanner!
+
+## Audit Scanner?
+
+Up until the Audit Scanner, Kubewarden was strictly a Dynamic Admission
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:13]:
```suggestion
Up until the release of Audit Scanner, Kubewarden was strictly a Dynamic Admission
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+the cluster resources compliance. The new Audit Scanner module is called via a
+Kubernetes `Cronjob`, which runs the jobs that check the cluster resources.
+
+#### Ok, but how does it work?
+
+To make that happen, the Audit Scanner looks at all the policies deployed, and which
+cluster resources are covered by the policies. It then build tuples of `(policy,
+synthetic_admission_request)` that match those, simulating operations to the resources.
+It then iterates through the tuples, sending their respective PolicyServers the
+`policy` and `syntethic_admission_request` to evaluate.
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:32]:
```suggestion
`policy` and `synthetic_admission_request` to evaluate.
```

File: content/blog/2023/09/audit-scanner-feature.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+
+Notice how the `synthetic_policy_request`s don't hit the Kubernetes API server;
+there's no need, as no resource is truly being CREATEd, UPDATEd, or DELETEd.
+Instead, the `synthetic_policy_request` is consumed by the PolicyServer. But
+not on its usual `validate/` HTTP endpoint, but in a different `audit/`
+endpoint, exclusive for audit requests. This `audit/` endpoint evaluates the
+policy exactly the same way, and is instrumented as well, which provides
```
Comment (jhkrug) [inline content/blog/2023/09/audit-scanner-feature.md:39]:
```suggestion
policy in exactly the same way, and is instrumented as well, which provides
```

File: content/blog/2023/10/kubewarden-1.8-release.md (hunk 1 above)
Comment (viccuad) [reply content/blog/2023/10/kubewarden-1.8-release.md]: Better! done in https://github.com/kubewarden/kubewarden.io/pull/195/commits/da56fe45efb8b26278c20dfd6626d670a90559d9.

File: content/blog/2023/10/kubewarden-1.8-release.md (hunk 2 above)
Comment (viccuad) [reply content/blog/2023/10/kubewarden-1.8-release.md]: Better! done in https://github.com/kubewarden/kubewarden.io/pull/195/commits/da56fe45efb8b26278c20dfd6626d670a90559d9.

File: content/blog/2023/10/kubewarden-1.8-release.md (hunk 3 above)
Comment (viccuad) [reply content/blog/2023/10/kubewarden-1.8-release.md]: Better! done in https://github.com/kubewarden/kubewarden.io/pull/195/commits/da56fe45efb8b26278c20dfd6626d670a90559d9.

File: content/blog/2023/09/audit-scanner-feature.md
Comment (viccuad) [reply content/blog/2023/09/audit-scanner-feature.md:1]: Indeed, fixed. This happened because the other PR for the audit feature needed a change in the folder (from sept `09` to October `10`), and this PR was based in that folder change, to not recreate the folder. Somehow it didn't went well..

**Review by viccuad — COMMENTED** [review]
Blocking, this shouldn't go live until This shouldn't go live until 1.8.0 is GA..

**Comment by fabriziosestito** [comment]
@viccuad reminder to link the article: https://opentelemetry.io/blog/2023/jaeger-exporter-collector-migration/#switch-to-otlp-exporter

**Comment by viccuad** [comment]
> @viccuad reminder to link the article: https://opentelemetry.io/blog/2023/jaeger-exporter-collector-migration/#switch-to-otlp-exporter

Done with https://github.com/kubewarden/kubewarden.io/pull/195/commits/94ca139cf6bf1e5179c594d3ce76def42eebcd2b.


Please re-review.

**Review by flavio — APPROVED** [review]

**Comment by flavio** [comment]
Moving to blocked, waiting for the actual 1.8.0 release to happen

**Comment by viccuad** [comment]
please re-review, added small paragraph with major version bump notice.

File: content/blog/2023/10/kubewarden-1.8-release.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+chart `values.yaml`, which is uncommon.
+We think it simplifies the adoption of OTLP changes in the future.
+
+In addition, since `v0.85.0` OpenTelemetry Collector no longer includes
+exporters for the native Jaeger format. This change was
+prompted because Jaeger has support for OTLP. See the [announcement
+here](https://opentelemetry.io/blog/2023/jaeger-exporter-collector-migration).
+
+Hence we now take care of setting `exporters.otlp/jaeger` instead of
+`exporters.jaeger` the OpenTelemetryCollector CR we create. For the user, this
```
Comment (flavio) [inline content/blog/2023/10/kubewarden-1.8-release.md:44]:
```suggestion
Now the OpenTelemetryCollector resource created by the helm chart is configured
to use  `exporters.otlp/jaeger` instead of `exporters.jaeger`. For the user, this
```

**Review by flavio — CHANGES_REQUESTED** [review]
LGTM, I left one small suggestion

**Review by flavio — APPROVED** [review]

**Review by fabriziosestito — APPROVED** [review]

**Review by jhkrug — APPROVED** [review]
lgtm
